src/oatpp-wolfssl/Config.cpp

```diff
@@ -0,0 +1,119 @@
+/***************************************************************************
+ *
+ * Project         _____    __   ____   _      _
+ *                (  _  )  /__\ (_  _)_| |_  _| |_
+ *                 )(_)(  /(__)\  )( (_   _)(_   _)
+ *                (_____)(__)(__)(__)  |_|    |_|
+ *
+ *
+ * Copyright 2018-present, Leonid Stryzhevskyi <lganzzzo@gmail.com>
+ *                         Benedikt-Alexander Mokroß <bam@icognize.de>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ***************************************************************************/
+
+#include "Config.hpp"
+
+#include "oatpp/core/base/Environment.hpp"
+
+#include "wolfssl/ssl.h"
+
+#if defined(OATPP_WOLFSSL_DEBUG)
+namespace oatpp { namespace wolfssl {
+static void wolfsslDebug(const int logLevel, const char* const logMessage) {
+  OATPP_LOGD("[wolfssl]", "[%s] %s", logLevel, logMessage)
+}
+}}
+#endif
+
+namespace oatpp { namespace wolfssl {
+
+Config::Config(WOLFSSL_METHOD *protocolMethod) {
+  int res = wolfSSL_Init();
```

Comment:
Is it safe to call this method multiple times?
[wolfSSL_Init](https://www.wolfssl.com/doxygen/group__TLS.html#ga789ef74e34df659a62f06da2ea709737)

Reply:
I was thinking that to myself when I wrote it.
I will fix it.